store/sqlstore/channel_member_history_store.go

```diff
@@ -4,9 +4,7 @@
 package sqlstore
 
 import (
-	"fmt"
 	"net/http"
-
```

Comment:
Because you removed this empty line the order of the imports must be alphabetical. This is a "gofmt" restriction. My recommendation here would be, configure you editor to execute gofmt on each save.

Reply:
Aha I see now, I've ran "gofmt -w" on the file and pushed it 👍